### ISSUE TITLE
[bug] TypeError: s.replace is not a function

### ISSUE THREAD

**Issue by weicong** [kubb-project/kubb#163]
```
TypeError: s.replace is not a function
    at escapeString (D:\Workspace\examples\react-query\node_modules\typescript\lib\typescript.js:16192:14)
    at escapeNonAsciiString (D:\Workspace\examples\react-query\node_modules\typescript\lib\typescript.js:16195:9)
    at getLiteralText (D:\Workspace\examples\react-query\node_modules\typescript\lib\typescript.js:13588:24)
    at getLiteralTextOfNode (D:\Workspace\examples\react-query\node_modules\typescript\lib\typescript.js:113610:14)
    at emitLiteral (D:\Workspace\examples\react-query\node_modules\typescript\lib\typescript.js:110871:20)
    at pipelineEmitWithHintWorker (D:\Workspace\examples\react-query\node_modules\typescript\lib\typescript.js:110685:20)
    at pipelineEmitWithHint (D:\Workspace\examples\react-query\node_modules\typescript\lib\typescript.js:110322:9)
    at pipelineEmitWithComments (D:\Workspace\examples\react-query\node_modules\typescript\lib\typescript.js:114101:7)
    at pipelineEmit (D:\Workspace\examples\react-query\node_modules\typescript\lib\typescript.js:110271:7)
    at emitExpression (D:\Workspace\examples\react-query\node_modules\typescript\lib\typescript.js:110255:7)
```

kubb.config.ts
```ts
import { defineConfig } from '@kubb/core'
import createSwagger from '@kubb/swagger'
import createSwaggerTS from '@kubb/swagger-ts'
import createSwaggerTanstackQuery from '@kubb/swagger-tanstack-query'

export default defineConfig({
  root: '.',
  input: {
    //path: './petStore.yaml',
    //path: 'https://zwyunqismr.zgkw.cn/swagger/v1/swagger.json',
    path: 'https://demo.aspnetzero.com/swagger/v1/swagger.json',
  },
  output: {
    path: './src/gen',
    clean: true,
  },
  // hooks: {
  //   done: 'eslint --fix ./src/gen',
  // },
  logLevel: 'warn',
  plugins: [
    createSwagger({ output: false }),
    createSwaggerTS({ output: 'models' }),
    createSwaggerTanstackQuery({ output: './hooks', groupBy: 'tag', framework: 'react' }),
  ],
})
```

**Comment by stijnvanhulle** [comment]
Seems that we have no support for `x-enumNames` when using enums. I will create a PR to fix it and that will be added in the next beta release 👍🏼 .

Examples of the support of enums we need:
```
 AbpLoginResultType:
      enum:
        - 1
        - 2
        - 3
        - 4
        - 5
        - 6
        - 7
        - 8
        - 9
        - 10
      type: integer
      format: int32
      x-enumNames:
        - Success
        - InvalidUserNameOrEmailAddress
        - InvalidPassword
        - UserIsNotActive
        - InvalidTenancyName
        - TenantIsNotActive
        - UserEmailIsNotConfirmed
        - UnknownExternalLogin
        - LockedOut
        - UserPhoneNumberIsNotConfirmed
```

**Comment by stijnvanhulle** [comment]
Fixed in version `1.0.0-beta.7`
